packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -439,6 +439,34 @@ export function frDict<
     [k in K4]: T4;
   } & { [k in K5]: T5 } & { [k in K6]: T6 }
 >;
+export function frDict<
```

Comment:
I ended up not needing this after removing ``points``, but I have a feeling we'll want it soon again.